package.json

```diff
@@ -241,6 +242,7 @@
     "klaw": "^3.0.0",
     "livepush": "^3.5.0",
     "lodash": "^4.17.20",
+    "memoizee": "^0.4.14",
```

Comment:
`memoizee` and `@types/memoizee` are used in the added `os.ts`, but the dependencies were already included in the `npm-shrinkwrap.json` file, which was thus not modified.